libs/ui/lib/properties-table/PropertiesTable.tsx

```diff
@@ -45,7 +45,7 @@ PropertiesTable.Row = ({ label, children }: PropertiesTableRowProps) => (
     <span className="flex items-center">
       <Badge>{label}</Badge>
     </span>
-    <div className="flex items-center overflow-hidden whitespace-nowrap pr-4 text-sans-md text-secondary">
+    <div className="flex h-11 items-center overflow-hidden whitespace-nowrap pr-4 text-sans-md text-secondary">
```

Comment:
We were getting a flash of movement when the skeleton switched to the text since they're not the exact same height. This fixes that and any future issues where the content of the row is not the same height as `text-sans-md`.